main.go

```diff
@@ -292,10 +292,8 @@ func main() {
 		}
 	}()
 
-	// Send the worker shutdown signal for each worker thread to read
-	workerShutdownChan <- true
-	workerShutdownChan <- true
-	workerShutdownChan <- true
+	// Close the channel, signalling to the worker threads to shutdown.
+	close(workerShutdownChan)
```

Comment:
Always was strange seeing the triple channel close :)

Reply:
Yeah... sometimes something starts small and quickly grows into piles of bad practice ;)